components/AppBar/examples/supplemental/SimpleComposedTableViewController.swift

```diff
@@ -0,0 +1,83 @@
+/// A simple table view controller which is manually composed.
+///
+/// This view controller is meaningfully *not* a subclass of UITableViewController and its view is
+/// not a UIScrollView subclass. This avoids logic which might attempt to be smart by detecting
+/// these cases.
+class SimpleComposedTableViewController: UIViewController {
```

Comment:
Please import UIKit.

Logs:
```
components/AppBar/examples/supplemental/SimpleComposedTableViewController.swift:6:42: error: use of undeclared type 'UIViewController'
class SimpleComposedTableViewController: UIViewController {
                                         ^~~~~~~~~~~~~~~~
components/AppBar/examples/supplemental/SimpleComposedTableViewController.swift:36:46: error: use of undeclared type 'UITableViewDataSource'
extension SimpleComposedTableViewController: UITableViewDataSource {
                                             ^~~~~~~~~~~~~~~~~~~~~
components/AppBar/examples/supplemental/SimpleComposedTableViewController.swift:47:46: error: use of undeclared type 'UITableViewDelegate'
extension SimpleComposedTableViewController: UITableViewDelegate {
                                             ^~~~~~~~~~~~~~~~~~~
components/AppBar/examples/supplemental/SimpleComposedTableViewController.swift:13:34: error: use of undeclared type 'NSCoder'
  required init?(coder aDecoder: NSCoder) {
                                 ^~~~~~~
components/AppBar/examples/supplemental/SimpleComposedTableViewController.swift:19:19: error: use of undeclared type 'MDCFlexibleHeaderView'
  var headerView: MDCFlexibleHeaderView?
                  ^~~~~~~~~~~~~~~~~~~~~
components/AppBar/examples/supplemental/SimpleComposedTableViewController.swift:21:17: error: method does not override any method from its superclass
  override func viewDidLoad() {
  ~~~~~~~~      ^
```

Reply:
Done. IDK why my Xcode was cool with missing imports, but it was.

Reply:
bazel is much pickier about enforcing import statements than Xcode + CocoaPods, so these errors unfortunately only pop up in our CI or from a local run of bazel build.